src/sentry/static/sentry/app/components/events/sdkUpdates.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
+
+import {t} from 'app/locale';
+
+class EventSdkUpdateSuggestion extends React.Component {
+  static propTypes = {
+    suggestion: PropTypes.object.isRequired,
+  };
+
+  getSuggestionTitle() {
+    const {suggestion} = this.props;
+    switch (suggestion.type) {
+      case 'updateSdk':
+        return <a href={suggestion.sdkUrl}>{t('update your SDK')}</a>;
+      case 'changeSdk':
+        return (
+          <a href={suggestion.sdkUrl}>
+            {t("migrate to the '%s' SDK", suggestion.newSdkName)}
+          </a>
+        );
+      case 'enableIntegration':
+        return (
+          <a href={suggestion.integrationUrl}>
+            {t("enable the '%s' integration", suggestion.integrationName)}
+          </a>
+        );
+      default:
+        return null;
+    }
+  }
+
+  render() {
+    const {suggestion} = this.props;
+    const title = this.getSuggestionTitle();
+
+    if (suggestion.enables.length == 0) {
+      return title;
+    }
+
+    return (
+      <span>
+        {title}
+        {t(' so you can: ')}
+        <ul>
+          {suggestion.enables.map((suggestion2, i) => {
+            return (
+              <li key={suggestion2}>
+                <EventSdkUpdateSuggestion suggestion={suggestion2} />
+              </li>
+            );
+          })}
+        </ul>
+      </span>
+    );
+  }
+}
+
+class EventSdkUpdates extends React.Component {
+  static propTypes = {
+    event: SentryTypes.Event.isRequired,
+  };
+
+  render() {
+    const {event} = this.props;
+    const data = event.sdkUpdates;
+
+    return (
+      <div>
+        {data.map(suggestion => {
+          return (
+            <div className="alert-block alert-info box row" key={suggestion}>
```

Comment:
`key` cannot be an object, has to be string/int and must be unique

Reply:
Both are now a sorted json dump of its properties. I don't expect this component to refresh, and I don't expect the backend to suggest the same thing twice.

Reply:
I would just use its array index then instead - it's possible it can cause odd behavior passing it an object (e.g. if it tries to cast it as a string and you end up with duplicate keys). It's not necessarily only about this component re-rendering, but something higher up in the component tree could re-render and cause this to re-render as well.